packages/web-components/src/components/feature-card/feature-card.ts

```diff
@@ -24,7 +24,7 @@ const { stablePrefix: ddsPrefix } = ddsSettings;
  * @element dds-feature-card
  */
 @customElement(`${ddsPrefix}-feature-card`)
```

Comment:
I think this `@customElement` declaration should be removed in favor of the one added below on line 62.